src/deployable_example.cpp

```diff
@@ -298,26 +298,41 @@ const char index_html[] PROGMEM = R"rawliteral(
   </head><body>
   <form action="/config">
     WiFi SSID: <input type="text" name="ssid" value="%ssid%">
-    WiFi Password: <input type="text" name="pw" value="%password%">
+    WiFi Password: <input type="password" name="pw" value="%password%"> 
+    API Key: <input type="password" name="api_key" value="%api_key%">
+    Autoconfig: <input type="checkbox" id="autoconfig" name="autoconfig" value="true" onchange="toggleAutoConfig()">
+    <div id="autoConfigMessage" style="display:none;" >
+      <p>If autoconfig is checked, all of the settings below will be ignored and get updated automatically.</p>
+    </div>
     Detector Id: <input type="text" name="det_id" value="%det_id%">
-    API Key: <input type="text" name="api_key" value="%api_key%">
     Query Delay (seconds): <input type="text" name="query_delay" value="%query_delay%">
     Endpoint: <input type="text" name="endpoint" value="%endpoint%">
     Target Confidence: <input type="text" name="tConf" value="%tConf%">
-    Motion Alpha (float between 0 and 1): <input type="text" name="mot_a" value="%mot_a%">
-    Motion Beta (float between 0 and 1): <input type="text" name="mot_b" value="%mot_b%">
-    Stacklight UUID: <input type="text" name="sl_uuid" value="%sl_uuid%">
-    Slack URL: <input type="text" name="slack_url" value="%slack_url%">
-    Email: <input type="text" name="email" value="%email%">
-    Email Endpoint: <input type="text" name="email_endpoint" value="%email_endpoint%">
-    Email Key: <input type="text" name="email_key" value="%email_key%">
-    Email Host: <input type="text" name="email_host" value="%email_host%">
-    Twilio SID: <input type="text" name="twilio_sid" value="%twilio_sid%">
-    Twilio Token: <input type="text" name="twilio_token" value="%twilio_token%">
-    Twilio Number: <input type="text" name="twilio_number" value="%twilio_number%">
-    Twilio Recipient: <input type="text" name="twilio_recipient" value="%twilio_recipient%">
```

Comment:
Are we intentionally killing support for notifications off the camera?

Reply:
Yes I killed the support for notifications because it is currently supported through the web app.